sh/sh.bzl

```diff
@@ -17,16 +17,6 @@ ShBinariesInfo = provider(
 
 _WINDOWS_EXE_EXTENSIONS = [".exe", ".cmd", ".bat", ".ps1"]
 
-def _normalize_label(label):
```

Comment:
The normalization was moved to the unittests in order to keep the core library clean.